runtime-modules/project-token/Cargo.toml

```
@@ -1,33 +1,35 @@
 [package]
 name = "pallet-project-token"
-version = '0.1.0'
+version = '5.0.0'
```

Incorrect update from `0.1.0` to `5.0.0`.
```suggestion
version = '1.0.0'
```
_I suggest sticking to `1.0.0`, since we never actually used this module on any testnet yet_

---

That makes sense of course, but since we never publish anything to a remote crates registry or use our crates in other projects, it doesn't really make a difference :) But I will commit suggestion

---

updated in 27f693078d3a90bd676a3f774f4b5ca837382b99